apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx

```diff
@@ -54,73 +54,69 @@ export const DelegationConfirmation = () => {
 
   return (
     <View style={styles.container}>
```

Comment:
is it a screen? if it is it should be wrapped within SafeAreaView.

Reply:
Maybe this is the issue?

Reply:
just checking and unfortunately the issue still persists

Reply:
also I tested using paddings instead of margings

Reply:
The problem is that the button stays behind the keyboard until you click to edit the input. When you start editing, the button goes into place.

I tried to force the focus but it didn't work either.